src/Microsoft.DotNet.Build.CloudTestTasks/AzureBlobLease.cs

```diff
@@ -29,7 +29,7 @@ public partial class AzureBlobLease
         private string _accountName;
         private string _accountKey;
         private Microsoft.Build.Utilities.TaskLoggingHelper _log;
-        private string _leaseId;
+        public string _leaseId { get; private set; }
```

Comment:
Properties are generally capitalized, and no '_'.  Why turn this into a property? I don't see it referenced anywhere.

Reply:
I want to use the same instance of AzureBlobLease to acquire and renew the leaseid, and be able to pass it to the header of subsequent requests.